composer.json

```diff
@@ -35,8 +35,6 @@
         "symfony/framework-bundle": "^4.4 || ^5.4 || ^6.0",
         "symfony/http-foundation": "^4.4 || ^5.4 || ^6.0",
         "symfony/http-kernel": "^4.4 || ^5.4 || ^6.0",
-        "symfony/mailer": "^4.4 || ^5.4 || ^6.0",
-        "symfony/mime": "^4.4.10 || ^5.4 || ^6.0",
```

Comment:
@VincentLanglet i'm unsure, is this a BC Break?

updating with composer would remove the dependency and composer would remove the packages?
which then causes a problem that the current default config (and what the user might have configured)
are still looking for the mailer packages? (i mean it probably would cause problems with cache:clear after composer update, so the user should see the problem directly?)

Reply:
Friendly ping @jordisala1991.
I don't remember where but I think we already did such change (which an upgrade note).

The other option would be to keep those dependencies and only remove them in 6.x branch

Reply:
Since you have to configure mailer, it should be a dep you have when installing this bundle. IMO we can try moving it to require-dev